codes/data/italy/italy-download-cases-merge-policies.py

```diff
@@ -510,54 +608,63 @@ def impute_cumulative_df(df, src_col, dst_col, groupby_col):
         adm1_cases[policy_name + popweighted_suffix] = 0
         adm2_cases[policy_name + popweighted_suffix] = 0
 
-def assign_policy_variables(adm_cases, policy_on_mask, policy, partial, perc_affected):
```

Comment:
I assume these are ITA-specific? If any of them are not, we could think about moving these to a `codes/policy.py` module or something like that. If they're pretty specific, just resolve this comment and leave them as-is

Reply:
I think this is a good idea. I spent a minute thinking about how to abstract this out yesterday and gave up because the policy_on_mask depends on the adm-level, which makes things a little complicated, definitely can be done though. Maybe for another PR?

Reply:
sounds good! will resolve

Reply:
@dpa9694 will you create an issue for this task?